src/components/containers/fieldset/Fieldset.js

```diff
@@ -27,20 +27,20 @@ const StyledFieldset = styled.fieldset`
   padding: 0;
 `;
 
-function Fieldset({ additionalLegendClasses, legendText, children, theme }) {
+function Fieldset({ additionalLegendClasses, legendContent, children, theme }) {
   return (
     <ThemeProvider theme={theme}>
       <StyledFieldset>
-        {renderLegend(legendText, additionalLegendClasses)}
+        {renderLegend(legendContent, additionalLegendClasses)}
         {children}
       </StyledFieldset>
     </ThemeProvider>
   );
 }
 
 Fieldset.propTypes = {
-  /** Determine whether or not to add a legend and what text to display  */
-  legendText: PropTypes.string,
+  /** Determine whether or not to add a legend and what content to display  */
+  legendContent: PropTypes.any,
```

Comment:
I do think `node` would be better, as it is more explicit about being renderable than `any`.